packages/quantic/force-app/examples/main/lwc/exampleLayout/exampleLayout.html

```diff
@@ -4,11 +4,11 @@
 
   <div class="slds-grid slds-wrap slds-gutters_direct slds-m-top_large">
     <div class="slds-col slds-size_1-of-1 slds-medium-size_1-of-2 slds-large-size_1-of-2">
-      <lightning-card title="Configuration">
+      <c-quantic-card-container title="Configuration">
```

Comment:
The SLDS changed their css for the lightning-card so our example layout was lacking some padding.